.github/workflows/dapr-perf.yml

```diff
@@ -462,11 +462,18 @@ jobs:
           #TODO: .json suffix can be removed from artifact name after test analytics scripts are updated
           name: test_perf.json
           path: ${{ env.TEST_OUTPUT_FILE_PREFIX }}_perf*.*
-      - name: Show test summary
+      - name: Add job test summary
         if: always()
         uses: test-summary/action@v2
         with:
           paths: ${{ env.TEST_OUTPUT_FILE_PREFIX }}_perf*.xml
+      - name: Add job test outputs
```

Comment:
Please don't run JS files as GitHub scripts unless they need access to the GitHub token, also for security reasons...

May be enough to just add `#!/usr/bin/env node` and then use this as a script (don't forget to `chmod +x` before committing the script).

Example: https://github.com/dapr/dapr/blob/master/.github/workflows/dapr.yml#L140-L142

Bonus: this allows you to name your file `.mjs` and then use ES imports and top-level await.

Reply:
The security implication is that scripts that run via the github-script action have access to the GitHub APIs pre-authorized and can make API calls that impact the repo. There are many cases where that's necessary, but if you don't (like in this case), it would be best to just run the script as a Node.js one with the `env node` hashbang.

Using `@actions/core` and `@actions/glob` would require github-script (likely), but I don't see them being used right now? Please correct me if I'm wrong.

Reply:
[Yes, I'm using them](https://github.com/dapr/dapr/pull/5856/files#diff-aed9855ec5ff916d6ced3a125bd9b76551a22febf05b7e6d4bafd9222eb26dfdR476)